packages/graphql/src/link.js

```diff
@@ -1,17 +1,27 @@
+import Observable from 'zen-observable'
```

Comment:
Does zen-observable need to be added to package.json ?

Reply:
Shouldn't be necessary unless you want to be explicit.  It's already a dependency of apollo.

Reply:
sounds good.